src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1410,73 +1410,72 @@ string[] linkingTargetFields
-            DisplayName = "API path prefix containing reserved characters /+&,")]
+            DisplayName = "GraphQL path prefix containing reserved characters /+&,")]
         [DataRow("/@)", "", true, ApiType.GraphQL, true,
-            DisplayName = "API path prefix containing reserved characters /@)")]
-        [DataRow("", "path prefix cannot be null or empty.", false, ApiType.GraphQL, true,
-            DisplayName = "Empty API path prefix.")]
-        [DataRow(null, "path prefix cannot be null or empty.", false, ApiType.GraphQL, true,
-            DisplayName = "Null API path prefix.")]
-        [DataRow("?", "path should start with a '/'.", false, ApiType.GraphQL, true,
-            DisplayName = "API path prefix not starting with forward slash.")]
+            DisplayName = "GraphQL path prefix containing reserved characters /@)")]
+        [DataRow("", "GraphQL path cannot be null or empty.", false, ApiType.GraphQL, true,
+            DisplayName = "Empty GraphQL path prefix.")]
+        [DataRow(null, "GraphQL path cannot be null or empty.", false, ApiType.GraphQL, true,
+            DisplayName = "Null GraphQL path prefix.")]
+        [DataRow("?", "GraphQL path should start with a '/'.", false, ApiType.GraphQL, true,
+            DisplayName = "GraphQL path not starting with forward slash.")]
         [DataRow("/-api", null, false, ApiType.GraphQL, false,
-            DisplayName = "API path prefix containing hyphen (-)")]
+            DisplayName = "GraphQL path containing hyphen (-)")]
         [DataRow("/api path", null, false, ApiType.GraphQL, false,
-            DisplayName = "API path prefix containing space in between")]
+            DisplayName = "GraphQL path prefix containing space in between")]
         [DataRow("/ apipath", null, false, ApiType.REST, false,
-            DisplayName = "API path prefix containing space at the start")]
+            DisplayName = "REST path containing space at the start")]
         [DataRow("/ api_path", null, false, ApiType.GraphQL, false,
-            DisplayName = "API path prefix containing space at the start and underscore in between.")]
-        [DataRow("/", null, false, ApiType.GraphQL, false,
-            DisplayName = "API path containing only a forward slash.")]
-        public void ValidateApiPathIsWellFormed(
-            string apiPathPrefix,
+            DisplayName = "GraphQL path prefix containing space at the start and underscore in between.")]
+        [DataRow("/", null, false, ApiType.REST, false,
+            DisplayName = "REST path containing only a forward slash.")]
+        public void ValidateApiURIsAreWellFormed(
+            string apiPath,
             string expectedErrorMessage,
             bool pathContainsReservedCharacters,
             ApiType apiType,
             bool expectError)
         {
-            ValidateRestAndGraphQLPathIsWellFormed(
-                apiPathPrefix,
+            ValidateRestAndGraphQLURIIsWellFormed(
```

Comment:
Are there any other callers of your validation logic (`ValidateRestAndGraphQLURIIsWellFormed()`)? I don't follow why a separate function is needed here

Reply:
This wasn't a change from this PR. But yeah, there are no other callers and this function can be removed. Thanks!